src/gp_activetable.h

```diff
@@ -37,6 +37,12 @@ typedef struct DiskQuotaActiveTableEntry
 	Size tablesize;
 } DiskQuotaActiveTableEntry;
 
+typedef struct ActiveTableEntryCombined
+{
+	Oid  reloid;
```

Comment:
Why is it necessary to save reloid in the entry? Can we get reloid from the key?